src/Razor/src/Microsoft.VisualStudio.Editor.Razor/RazorDirectiveCompletionProvider.cs

```diff
@@ -109,15 +109,15 @@ public override Task ProvideCompletionsAsync(CompletionContext context)
                 return Task.CompletedTask;
             }
 
-            var result = AddCompletionItems(context);
+            var result = AddCompletionItemsAsync(context);
 
             return result;
         }
 
         // We do not want this inlined because the work done in this method requires Razor.Workspaces and Razor.Language assemblies.
         // If those two assemblies were to load you'd have them load in every C# editor completion scenario.
         [MethodImpl(MethodImplOptions.NoInlining)]
-        private Task AddCompletionItems(CompletionContext context)
+        private Task AddCompletionItemsAsync(CompletionContext context)
```

Comment:
We should turn off this rule. I don't care what VS threading thinks about our method names.